### PR TITLE
default pr-branch to subName + random string

### PR DESCRIPTION
**Pull request by aclevername**
### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

**Comment by Skarlso** [comment]
Is the change to dependencies/profiles intentional?

**Comment by aclevername** [comment]
> Is the change to dependencies/profiles intentional? I don't see a modules update to a new version of it.

I just noticed it was behind and bumped, no actual reason. Happy to remove it if you'd like 😃

**Review by Skarlso — APPROVED** [review]
LGTM

File: cmd/pctl/install.go [hunk 1]
```diff
@@ -173,6 +171,9 @@ func createPullRequest(c *cli.Context) error {
 	if repo == "" {
 		return errors.New("repo must be defined if create-pr is true")
 	}
+	if branch == "" {
+		branch = c.String("subscription-name") + "-" + uuid.NewString()[0:6]
```
Comment (Skarlso) [inline cmd/pctl/install.go:175]:
```suggestion
		branch = c.String("subscription-name") + "-" + uuid.NewString()[:6]
```

Comment (aclevername) [reply cmd/pctl/install.go:175]: oooo nice!

**Pull request merged**